LiteCore/Support/Logging.cc

```diff
@@ -57,6 +57,8 @@ namespace litecore {
     static LogDomain::Callback_t sCallback = LogDomain::defaultCallback;
     static bool sCallbackPreformatted = false;
     LogLevel LogDomain::sFileMinLevel = LogLevel::None;
+    unsigned LogDomain::slastObjRef {0};
+    map<unsigned, string> LogDomain::sobjNames;
```

Comment:
Nitpick: Please capitalize as `sObjNames`.

Reply:
That was a straight up typo!  I would normally do that and I don't think it's a nit pick.